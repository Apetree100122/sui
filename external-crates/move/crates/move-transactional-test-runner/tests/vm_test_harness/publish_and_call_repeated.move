//# publish-and-call --call 0x42::m::a 0 0 0 --call 0x42::m::b vector[1,2,3] --call 0x42::m::a 0 1 3 --call 0x42::m::b vector[1,2,3]
module 0x42::m {
    fun a(n: u64, m: u64, o: u64): u64 {
        n + m + o
    }

    fun b(_y: vector<u64>): u64 {
        10
    }
}
