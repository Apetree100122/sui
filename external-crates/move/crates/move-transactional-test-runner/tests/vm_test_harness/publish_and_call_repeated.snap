---
source: crates/move-transactional-test-runner/src/framework.rs
---
processed 1 task

task 0, lines 1-10:
//# publish-and-call --call 0x42::m::a 0 0 0 --call 0x42::m::b vector[1,2,3] --call 0x42::m::a 0 1 3 --call 0x42::m::b vector[1,2,3]
return values: 0
return values: 10
return values: 4
return values: 10
